src/components/Footer/BottomBar.tsx

```diff
@@ -1,33 +1,34 @@
 import React from 'react';
-import {AppBar, Box, Link, Toolbar, Typography} from "@mui/material";
+import {AppBar, Link, ThemeProvider, Toolbar, Typography} from "@mui/material";
+import {drtTheme} from "../../index";
 
 export interface BottomBarProps {
-  title: string;
   email: string;
-  clickAccessibility: () => void;
+  onClickAccessibilityStatement: () => void;
   url: string;
 }
 
-export const BottomBar = ({title, email, clickAccessibility, url}: BottomBarProps) => {
+export const BottomBar = ({email, onClickAccessibilityStatement, url}: BottomBarProps) => {
   return (
-    <AppBar position="static" color="default" sx={{top: 'auto', bottom: 0}}>
-      <Toolbar sx={{
-        display: 'flex',
-        gap: '12px',
-        flexDirection: {xs: 'column', sm: 'row'},
-        alignItems: {xs: 'flex-start', sm: 'center'}
-      }}>
-        <Typography variant="body1">Support links:</Typography>
-        <Link href={`mailto:${email}`} target="_blank" underline="always">Email us</Link>
-        <Link underline="always" onClick={() => {
-          window.location.hash = 'accessibility';
-          clickAccessibility();
-        }} title={`${title}`}>
-          Accessibility statement
-        </Link>
-        <Link href={`${url}`} target="_blank" underline="always">Give feedback</Link>
-      </Toolbar>
-    </AppBar>
+    <ThemeProvider theme={drtTheme}>
```

Comment:
Is this because of a scala.js thing? ThemeProvider should normally be passed in once rather than at a component level.

Reply:
yeah it is scala js which cause theme to override .

Reply:
Created facade and removed theme from here .